limits.h

```diff
@@ -22,6 +22,9 @@
 #ifndef limits_h
 #define limits_h 
 
+// Size of the flag in mm
```

Comment:
Again, maybe a more descriptive comment.

// Largest possible distance to travel during pull off motion.  Defined by the a largest 
// homing flag in the system.  This is the Y flag which has a length of just under 40mm